support-data-rest/src/main/java/org/uniprot/api/support/data/taxonomy/request/GetByTaxonIdsRequest.java

```diff
@@ -0,0 +1,95 @@
+package org.uniprot.api.support.data.taxonomy.request;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.PositiveOrZero;
+
+import lombok.Data;
+
+import org.springframework.http.MediaType;
+import org.uniprot.api.rest.request.ReturnFieldMetaReaderImpl;
+import org.uniprot.api.rest.request.SearchRequest;
+import org.uniprot.api.rest.validation.*;
+import org.uniprot.api.support.data.taxonomy.repository.TaxonomyFacetConfig;
+import org.uniprot.core.util.Utils;
+import org.uniprot.store.config.UniProtDataType;
+
+import uk.ac.ebi.uniprot.openapi.extension.ModelFieldMeta;
+import io.swagger.v3.oas.annotations.Parameter;
+
+/**
+ * @author lgonzales
+ * @since 17/09/2020
+ */
+@Data
+public class GetByTaxonIdsRequest implements SearchRequest {
+
+    private static final String TAXONOMY_ID_LIST_REGEX = "^\\d+(?:,\\d+)*$";
+
```

Comment:
Please make this String constant hidden otherwise it will be generated in swagger doc.
@Parameter(hidden = true)